internal_metrics/metrics.yaml

```diff
@@ -0,0 +1,253 @@
+  brief: Ebpf collector log count
+  description: |
+     Ebpf collector log count
+  metric_type: count
+  title: ebpf_net.collector_log_count
+
+ebpf_net.time_since_last_message_ns:
+  brief: Time since last message in ns.
+  description: |
+    Time since last message in ns.
+  metric_type: counter
+  title: ebpf_net.time_since_last_message_ns
+
+ebpf_net.bpf_log:
+  brief: EBPF log count.
+  description: |
+     EBPF log count. This is enabled by default.
+  metric_type: counter
+  title: ebpf_net.bpf_log
+
+ebpf_net.codetiming_count:
+  brief: The number of codetiming blocks.
+  description: |
+     The number of codetiming blocks.
+  metric_type: counter
+  title: ebpf_net.codetiming_count
+
+ebpf_net.message:
+  brief: Message count.
+  description: |
+     Message count for the prior 30 seconds.
+  metric_type: counter
+  title: ebpf_net.message
+
+ebpf_net.otlp_grpc.bytes_sent:
+  brief: Total number of OTLP bytes in the prior 30 seconds.
+  description: |
+    Total number of OTLP bytes in the prior 30 seconds.  This is enabled by default.
+  metric_type: counter
+  title: ebpf_net.otlp_grpc.bytes_sent
+
+ebpf_net.otlp_grpc.bytes_failed:
+  brief: Total number of OTLP bytes failed in the prior 30 seconds.
+  description: |
+    Total number of OTLP bytes failed in the prior 30 seconds. This is enabled by default.
+  metric_type: counter
+  title: ebpf_net.otlp_grpc.bytes_failed
+
+ebpf_net.pipeline_message_error:
+  brief:  Pipeline message error.
+  description: |
+    Total number of pipeline message errors.
+  metric_type: counter
+  title:  ebpf_net.pipeline_message_error
+
+ebpf_net.pipeline_metric_bytes_written:
+  brief:  Total number of pipeline metric bytes written.
+  description: |
+    Total number of pipeline metric bytes written.
+  metric_type: counter
+  title:  ebpf_net.pipeline_metric_bytes_written
+
+ebpf_net.span_utilization_max:
+  brief:  Maximum span utlization
+  description: |
+    Maximum span utlization.
+  metric_type: counter
+  title:  ebpf_net.span_utilization_max
+
+ebpf_net.client_handle_pool_fraction:
+  brief:  Client handle pool fraction.
+  description: |
+    Client handle pool fraction.
+  metric_type: counter
+  title:  ebpf_net.client_handle_pool_fraction
+
+ebpf_net.rpc_latency_ns:
+  brief:  RPC latency in ns.
+  description: |
+    Remote Procedure Calls (RPC) latencies in ns
+  metric_type: counter
+  title:  ebpf_net.rpc_latency_ns
+
+ebpf_net.agg_root_truncation:
+  brief:  Agg root truncation.
+  description: |
+    Total number of Aggregation root trunctions in Agg app.
+  title:  ebpf_net.agg_root_truncation
+
+ebpf_net.clock_offset_ns:
+  brief:  Clock offset in ns.
+  description: |
+    Clock offset in nanoseconds.
+  metric_type: counter
+  title:  ebpf_net.clock_offset_ns
+
+ebpf_net.otlp_grpc.unknown_response_tags:
+  brief:  Total number of unknown OTLP grpc response tags.
+  description: |
+    Total number of unknown OTLP grpc response tags. This is enabled by default.
```

Comment:
Could add something like "A non-zero value would indicate a logic error in the code or a configuration error."